src/libs/scene_utils/samples/cesium_sample/sample_utils.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 2022
+// SPDX-License-Identifier: Apache-2.0
+
+import minimist from 'minimist';
+
+export interface CesiumSampleArguments {
+  workspaceId: string;
+  sceneId: string;
+  assetFilePath?: string;
+  cesiumAccessToken?: string;
+  cesiumAssetId?: string;
+}
+
+export const help = () => {
+  console.log(`Configure the AWS credentials and AWS region in your environment by setting env variables:
+    * AWS_ACCESS_KEY
+    * AWS_SECRET_ACCESS_KEY
+    * AWS_SESSION_TOKEN
+    * AWS_REGION (e.g. us-east-1)
+  
+  Usage: 
+    
+    arguments:
+      --workspaceId         REQUIRED
```

Comment:
since workspaces are region-specific, personally I think is easier/less-error-prone to allow passing in the region as an argument (similar to AWS CLI) instead of via environment variable

Reply:
I'm also about to add an optional argument to enable draco compression on an imported asset. I can add another required argument for the region, just not sure if it's too many arguments.

Reply:
Discussed offline: for now we will not have customers pass in the region as an argument. Will make it clear in the README that `AWS_REGION` must be set as a prerequisite along with the AWS credentials env vars.